### PR TITLE
New footer (again)

### PR DESCRIPTION
**Pull request by ajb**

### CODE REVIEW

**Comment by ajb** [comment]
Can we implement the sticky footer _without_ the layout fixes?

**Comment by jrubenoff** [comment]
Done. Forgot to mention that we need to add Autoprefixer to the build process here at some point... older browsers won't receive the sticky footer at the moment.

**Comment by ajb** [comment]
Thanks, I'll let @aviv review and merge since I messed up earlier. Will create an issue for autoprefixer.

**Comment by aviv** [comment]
Am I missing something?

**Comment by jrubenoff** [comment]
... I guess we should add Autoprefixer before merging this in.

**Comment by ajb** [comment]
Will this degrade gracefully?

File: contents/styles/knowledge_base.scss [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
 
 // Base
 
 body {
-  background-color: $footerBackground;
+  padding-top: $navHeight;
+  // Sticky footer
+  display: flex;
+  min-height: 100vh;
+  flex-direction: collumn;
```
Comment (ajb) [inline contents/styles/knowledge_base.scss]: @aviv @jrubenoff: Actually, it's just this typo.

**Comment by jrubenoff** [comment]
This could be a bug I need to fix, but older browsers should just ignore the rules. I'm assuming for now that this is a case where Aviv's browser is only recognizing some flexbox styles because it only supports prefixed rules.

**Comment by aviv** [comment]
Yep, it was that typo. Pushing up a fix (and adding autoprefixer) momentarily.

**Comment by ajb** [comment]
Here are our autoprefixer rules from Screendoor: https://github.com/dobtco/screendoor-v2/blob/master/config/autoprefixer.yml

**Comment by aviv** [comment]
OK seems to be working, as I'm seeing this in our CSS:

```css
body{padding-top:4.5rem;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;min-height:100vh;-webkit-box-orient:vertical;-webkit-box-direction:normal;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column}.content{-webkit-box-flex:1;-webkit-flex:1;-ms-flex:1;flex:1}
```

**Comment by ajb** [comment]
Looks great. It probably makes sense to browser test this before shipping. Give browserstack screenshots a tr?

**Comment by aviv** [comment]
Yep. Here's IE 10:

**Comment by aviv** [comment]
Will merge in the morning.

**Comment by ajb** [comment]
Again, that rebase was a big premature... I can no longer see how you fixed the problem. And can we do a suite of screenshot tests before merging, like I asked?

**Comment by ajb** [comment]
Looks good on Chrome, though.

**Comment by aviv** [comment]
Ah sorry. I just changed [these](https://github.com/dobtco/knowledge_base/pull/171/files#diff-9bab0a0007f393f6da3fc6ef2fdf8533R17) [two](https://github.com/dobtco/knowledge_base/pull/171/files#diff-9bab0a0007f393f6da3fc6ef2fdf8533R21) lines. Suite of screenshot tests == running screenshotter locally?

**Comment by ajb** [comment]
No - go to browserstack.com/screenshots, choose the appropriate browsers, use the local tunnel, and run it.

**Comment by aviv** [comment]
Home page: http://www.browserstack.com/screenshots/72dceb25557dd78105f4b2ad0d0fbb5cb5e3092b

Short article: http://www.browserstack.com/screenshots/374a59edf7c4c9a0f4a1f597283ca37f909de362

The header has some extra padding below it on really old browsers, but seems :white_check_mark: to me.

**Comment by ajb** [comment]
Cool, lgtm. Thanks for going the extra mile. :checkered_flag:

**Pull request merged**